src/js/common/schemaform/ObjectField.jsx

```diff
@@ -49,29 +36,13 @@ class ObjectField extends React.Component {
     this.onPropertyChange = this.onPropertyChange.bind(this);
     this.onPropertyBlur = this.onPropertyBlur.bind(this);
     this.isRequired = this.isRequired.bind(this);
-    // This runs a series of steps that order properties and then group them into
-    // expandable groups. If there are no expandable groups, then the end result of this
-    // will be an array of single item arrays
-    this.orderAndFilterProperties = _.flow(
-      properties => orderProperties(properties, _.get('ui:order', this.props.uiSchema)),
-      _.groupBy((item) => {
-        const expandUnderField = _.get([item, 'ui:options', 'expandUnder'], this.props.uiSchema);
-        return expandUnderField || item;
-      }),
-      _.values
-    );
+    this.SchemaField = pureWithDeepEquals(this.props.registry.fields.SchemaField);
```

Comment:
This resulted in something like a 50% improvement in the length of the key press event when typing in a form field.